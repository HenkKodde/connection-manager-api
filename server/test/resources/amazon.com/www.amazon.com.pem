0 s:/C=US/ST=Washington/L=Seattle/O=Amazon.com, Inc./CN=www.amazon.com
   i:/C=US/O=DigiCert Inc/CN=DigiCert Global CA G2
-----BEGIN CERTIFICATE-----
MIIHgDCCBmigAwIBAgIQC1QqOWBPlnhP1A/RBwvk2zANBgkqhkiG9w0BAQsFADBE
MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMR4wHAYDVQQDExVE
aWdpQ2VydCBHbG9iYWwgQ0EgRzIwHhcNMTkwMzI3MDAwMDAwWhcNMjAwMjI3MTIw
MDAwWjBoMQswCQYDVQQGEwJVUzETMBEGA1UECBMKV2FzaGluZ3RvbjEQMA4GA1UE
BxMHU2VhdHRsZTEZMBcGA1UEChMQQW1hem9uLmNvbSwgSW5jLjEXMBUGA1UEAxMO
d3d3LmFtYXpvbi5jb20wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQD5
S25hsY9ybVXCygMkKMvgS+PF7mPUva4T94jbT50foNbyUrx8qpiwHpp+sDf9g/1m
nZZgt6Hb6hPfeUnCVi/Z4ShzRyxMvLy9UX6mHmCJRZVXCyoi5IfIDtJB+ZiTPkNi
6RSSG7qsxqd/pBvJqLsQEBGfkI/aP8+frhjfCapucsCHtiRzixCtSGvhlURDHbiz
5RS75M9garMpuBoLojyYjUu44XG0Tq92C+VXjMlYdV2mK9VjimRaPxpqspBzlSUC
tZl8ZkiUSDOcBswyI5avY3+2NQMdGaNwznMjPJavxuytnKVJ2ESByJjid2hNBvgo
rF64IeQXsUIWPSavpLtlAgMBAAGjggRIMIIERDAfBgNVHSMEGDAWgBQkbist0GqS
UVElaQGqmkemiedAIDAdBgNVHQ4EFgQUBLYtrEQTu07u5zL2jmDkBCG3CoYwggGC
BgNVHREEggF5MIIBdYINeXAuYW1hem9uLmNvbYIMd3d3LmFtem4uY29tgg53d3cu
YW1hem9uLmNvbYINdXMuYW1hem9uLmNvbYIRdWVkYXRhLmFtYXpvbi5jb22CJXAt
eW8td3d3LWFtYXpvbi1jb20ta2FsaWFzLmFtYXpvbi5jb22CJXAteTMtd3d3LWFt
YXpvbi1jb20ta2FsaWFzLmFtYXpvbi5jb22CJXAtbnQtd3d3LWFtYXpvbi1jb20t
a2FsaWFzLmFtYXpvbi5jb22CFW9yaWdpbi13d3cuYW1hem9uLmNvbYIRaXBob25l
LmFtYXpvbi5jb22CEmh1ZGRsZXMuYW1hem9uLmNvbYIPaG9tZS5hbWF6b24uY29t
ghRjb3Jwb3JhdGUuYW1hem9uLmNvbYIRYnV5Ym94LmFtYXpvbi5jb22CIWJ1Y2tl
eWUtcmV0YWlsLXdlYnNpdGUuYW1hem9uLmNvbYIIYW16bi5jb22CCmFtYXpvbi5j
b20wDgYDVR0PAQH/BAQDAgWgMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcD
AjB3BgNVHR8EcDBuMDWgM6Axhi9odHRwOi8vY3JsMy5kaWdpY2VydC5jb20vRGln
aUNlcnRHbG9iYWxDQUcyLmNybDA1oDOgMYYvaHR0cDovL2NybDQuZGlnaWNlcnQu
Y29tL0RpZ2lDZXJ0R2xvYmFsQ0FHMi5jcmwwTAYDVR0gBEUwQzA3BglghkgBhv1s
AQEwKjAoBggrBgEFBQcCARYcaHR0cHM6Ly93d3cuZGlnaWNlcnQuY29tL0NQUzAI
BgZngQwBAgIwdAYIKwYBBQUHAQEEaDBmMCQGCCsGAQUFBzABhhhodHRwOi8vb2Nz
cC5kaWdpY2VydC5jb20wPgYIKwYBBQUHMAKGMmh0dHA6Ly9jYWNlcnRzLmRpZ2lj
ZXJ0LmNvbS9EaWdpQ2VydEdsb2JhbENBRzIuY3J0MAkGA1UdEwQCMAAwggEDBgor
BgEEAdZ5AgQCBIH0BIHxAO8AdQCkuQmQtBhYFIe7E6LMZ3AKPDWYBPkb37jjd80O
yA3cEAAAAWnAc+ECAAAEAwBGMEQCIGUDOX5O9o11/GHTfK0nkjSovX1kKtA6MvBy
6yT9ojzMAiAJCq5eOSqenmBwuns7yVEh5iIiXjJdKR9dcxs84O6CMgB2AId1v+dZ
fPiMQ5lfvfNu/1aNR1Y2/0q1YMG06v9eoIMPAAABacBz4h4AAAQDAEcwRQIgBpSC
yYRcjK+HdASo2B6r5JxplWfqJLd0ESWjd3yXcWQCIQD5NuV5gTSsVUdGX5+KGq0v
QHVp+pjUjhAjqu/hSzSU4DANBgkqhkiG9w0BAQsFAAOCAQEAqRzwW6qgdCMNTJMo
U7CFTOttXHjy81zUAId93IbxqVm3ny1m+Qa/kqleTyeMTfwudMdBLMZowH1Wnydh
u46gREHISAk/6YV9YC4ueVYNfZM8a15N0LwwsS0mQUzns3OYyE8SiYFXC4X3r9g6
gL1cvx6Bc/Tq7Wj4LEHkJIgSyf35GYWtCbQZiJif+Vdf6wZQLfcTv4mu+C2br4gt
yDn57bCyXQbl43PxsieXqPeAMjNnXkRE/4u0PbnB9JGbqYDGYQWqNdBFTqoczd6F
MMC3vCa1iX1WHJlkt7l1gAe/zurgbMDZopiSIwmEpUmJoWCAvEgQOD1v1mwCRhGE
bqQ9tg==
-----END CERTIFICATE-----
