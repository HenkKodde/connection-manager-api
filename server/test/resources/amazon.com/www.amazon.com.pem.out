Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            0b:54:2a:39:60:4f:96:78:4f:d4:0f:d1:07:0b:e4:db
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, O=DigiCert Inc, CN=DigiCert Global CA G2
        Validity
            Not Before: Mar 27 00:00:00 2019 GMT
            Not After : Feb 27 12:00:00 2020 GMT
        Subject: C=US, ST=Washington, L=Seattle, O=Amazon.com, Inc., CN=www.amazon.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:f9:4b:6e:61:b1:8f:72:6d:55:c2:ca:03:24:28:
                    cb:e0:4b:e3:c5:ee:63:d4:bd:ae:13:f7:88:db:4f:
                    9d:1f:a0:d6:f2:52:bc:7c:aa:98:b0:1e:9a:7e:b0:
                    37:fd:83:fd:66:9d:96:60:b7:a1:db:ea:13:df:79:
                    49:c2:56:2f:d9:e1:28:73:47:2c:4c:bc:bc:bd:51:
                    7e:a6:1e:60:89:45:95:57:0b:2a:22:e4:87:c8:0e:
                    d2:41:f9:98:93:3e:43:62:e9:14:92:1b:ba:ac:c6:
                    a7:7f:a4:1b:c9:a8:bb:10:10:11:9f:90:8f:da:3f:
                    cf:9f:ae:18:df:09:aa:6e:72:c0:87:b6:24:73:8b:
                    10:ad:48:6b:e1:95:44:43:1d:b8:b3:e5:14:bb:e4:
                    cf:60:6a:b3:29:b8:1a:0b:a2:3c:98:8d:4b:b8:e1:
                    71:b4:4e:af:76:0b:e5:57:8c:c9:58:75:5d:a6:2b:
                    d5:63:8a:64:5a:3f:1a:6a:b2:90:73:95:25:02:b5:
                    99:7c:66:48:94:48:33:9c:06:cc:32:23:96:af:63:
                    7f:b6:35:03:1d:19:a3:70:ce:73:23:3c:96:af:c6:
                    ec:ad:9c:a5:49:d8:44:81:c8:98:e2:77:68:4d:06:
                    f8:28:ac:5e:b8:21:e4:17:b1:42:16:3d:26:af:a4:
                    bb:65
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Authority Key Identifier: 
                keyid:24:6E:2B:2D:D0:6A:92:51:51:25:69:01:AA:9A:47:A6:89:E7:40:20

            X509v3 Subject Key Identifier: 
                04:B6:2D:AC:44:13:BB:4E:EE:E7:32:F6:8E:60:E4:04:21:B7:0A:86
            X509v3 Subject Alternative Name: 
                DNS:yp.amazon.com, DNS:www.amzn.com, DNS:www.amazon.com, DNS:us.amazon.com, DNS:uedata.amazon.com, DNS:p-yo-www-amazon-com-kalias.amazon.com, DNS:p-y3-www-amazon-com-kalias.amazon.com, DNS:p-nt-www-amazon-com-kalias.amazon.com, DNS:origin-www.amazon.com, DNS:iphone.amazon.com, DNS:huddles.amazon.com, DNS:home.amazon.com, DNS:corporate.amazon.com, DNS:buybox.amazon.com, DNS:buckeye-retail-website.amazon.com, DNS:amzn.com, DNS:amazon.com
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication, TLS Web Client Authentication
            X509v3 CRL Distribution Points: 

                Full Name:
                  URI:http://crl3.digicert.com/DigiCertGlobalCAG2.crl

                Full Name:
                  URI:http://crl4.digicert.com/DigiCertGlobalCAG2.crl

            X509v3 Certificate Policies: 
                Policy: 2.16.840.1.114412.1.1
                  CPS: https://www.digicert.com/CPS
                Policy: 2.23.140.1.2.2

            Authority Information Access: 
                OCSP - URI:http://ocsp.digicert.com
                CA Issuers - URI:http://cacerts.digicert.com/DigiCertGlobalCAG2.crt

            X509v3 Basic Constraints: 
                CA:FALSE
            1.3.6.1.4.1.11129.2.4.2: 
                ......u.......X......gp
<5.......w........i.s.......F0D. e.9~N..u.a.|.'.4..}d*.:2.r.$..<.. .
.^9*..`p.{;.Q!.""^2]).]s.<...2.v..u..Y|..C._..n.V.GV6.J.`....^......i.s.......G0E. .....\...t.......i.g.$.t.%.w|.qd.!..6.y.4.UGF_..../@ui.....#...K4..
    Signature Algorithm: sha256WithRSAEncryption
         a9:1c:f0:5b:aa:a0:74:23:0d:4c:93:28:53:b0:85:4c:eb:6d:
         5c:78:f2:f3:5c:d4:00:87:7d:dc:86:f1:a9:59:b7:9f:2d:66:
         f9:06:bf:92:a9:5e:4f:27:8c:4d:fc:2e:74:c7:41:2c:c6:68:
         c0:7d:56:9f:27:61:bb:8e:a0:44:41:c8:48:09:3f:e9:85:7d:
         60:2e:2e:79:56:0d:7d:93:3c:6b:5e:4d:d0:bc:30:b1:2d:26:
         41:4c:e7:b3:73:98:c8:4f:12:89:81:57:0b:85:f7:af:d8:3a:
         80:bd:5c:bf:1e:81:73:f4:ea:ed:68:f8:2c:41:e4:24:88:12:
         c9:fd:f9:19:85:ad:09:b4:19:88:98:9f:f9:57:5f:eb:06:50:
         2d:f7:13:bf:89:ae:f8:2d:9b:af:88:2d:c8:39:f9:ed:b0:b2:
         5d:06:e5:e3:73:f1:b2:27:97:a8:f7:80:32:33:67:5e:44:44:
         ff:8b:b4:3d:b9:c1:f4:91:9b:a9:80:c6:61:05:aa:35:d0:45:
         4e:aa:1c:cd:de:85:30:c0:b7:bc:26:b5:89:7d:56:1c:99:64:
         b7:b9:75:80:07:bf:ce:ea:e0:6c:c0:d9:a2:98:92:23:09:84:
         a5:49:89:a1:60:80:bc:48:10:38:3d:6f:d6:6c:02:46:11:84:
         6e:a4:3d:b6
